### ISSUE TITLE
Uncaught error with Modal from version 5.5.0

### ISSUE THREAD

**Issue by eeemarv** [bestguy/sveltestrap#381]
Hi, 
I'm using Sveltestrap in an Electronjs app and from Svelterstrap version 5.5.0 I get this fatal error when trying to open a Modal (nothing happens and app blocks):

```
Uncaught (in promise) RangeError: Maximum call stack size exceeded
    at getWindow (getWindow.js:1)
    at getComputedStyle (getComputedStyle.js:3)
    at getComputedStyle (getComputedStyle.js:3)
    at getComputedStyle (getComputedStyle.js:3)
    at getComputedStyle (getComputedStyle.js:3)
    at getComputedStyle (getComputedStyle.js:3)
    at getComputedStyle (getComputedStyle.js:3)
    at getComputedStyle (getComputedStyle.js:3)
    at getComputedStyle (getComputedStyle.js:3)
    at getComputedStyle (getComputedStyle.js:3)
```

In version 5.4.1-0 the modals are working fine. 

(node: '14.7.0)

**Comment by bestguy** [comment]
Thanks, hmm that's strange, let me see what may have changed here in those versions.

**Comment by eeemarv** [comment]
It seems it's fixed now when upgrading to version 5.7.0. Thanks!
